### PR TITLE
Add language about accommodating CC provider preference

### PR DESCRIPTION
**Pull request by jenniemc**
## Description
Add alert box with language about accommodating CC provider preference

## Screenshots
image

## Acceptance criteria
- [ ] alert box with language is displayed
- [ ] Change form title to "Mailing address line 1" and "Mailing address line 2"
- [ ] include the type of care in the preferred provider title

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/reducers/newAppointment.js [hunk 1]
```diff
@@ -722,7 +722,15 @@ export default function formReducer(state = initialState, action) {
     }
     case FORM_PAGE_COMMUNITY_CARE_PREFS_OPEN_SUCCEEDED: {
       let formData = state.data;
-      let initialSchema = action.schema;
+      const typeOfCare = getTypeOfCare(formData);
+      debugger;
```
Comment (narin) [inline src/applications/vaos/reducers/newAppointment.js:726]: You can remove this debugger

**Review by narin — COMMENTED** [review]
Looks good overall.  Just a comment to remove the debugger statement.  You also might want to push a minor change to re-trigger the jenkins build.

File: src/applications/vaos/containers/CommunityCarePreferencesPage.jsx [hunk 2]
```diff
@@ -127,6 +132,16 @@ const uiSchema = {
...
+      'ui:help': (
+        <AlertBox
+          status="info"
+          headline="We’ll try to schedule your appointment with your preferred community provider"
+        >
+          If we aren’t able to schedule this appointment with your preferred
+          provider, we’ll make every effort to schedule your appointment with
+          another community provider closest to your home.
+        </AlertBox>
+      ),
```
Comment (jenniemc) [inline src/applications/vaos/containers/CommunityCarePreferencesPage.jsx:144]: Hi @jbalboni , the layout for the alert message is to display at the end of communityCareProvider uiSchema.  Visually it is under the phone input text box, but inserting the alertbox within the phone uiSchema describes the wrong property. How can i have it display in the desired location and describe the communityCareProvider object?

Comment (jbalboni) [reply src/applications/vaos/containers/CommunityCarePreferencesPage.jsx:144]: Typically what we do in this situation is create an empty field, like this: https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/forms/common-patterns-for-building-forms#creating-a-block-of-text-with-no-associated-fields

In that example, you would put your component in place of the text used for `ui:description`.

Comment (jenniemc) [reply src/applications/vaos/containers/CommunityCarePreferencesPage.jsx:150]: Thanks, worked out perfectly

**Review by jbalboni — APPROVED** [review]
LGTM

**Pull request merged**